backend/src/test/java/net/vidux/camhub/discovery/CameraScanTest.java

```diff
@@ -0,0 +1,61 @@
+package net.vidux.camhub.discovery;
+
+import java.util.Set;
+import java.util.stream.Stream;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.MethodSource;
+
+class CameraScanTest {
+
+  @Test
+  void testViduxHelper() {
+    Assertions.assertDoesNotThrow(
```

Comment:
Ha a teszt dob egy kivételt, akkor egyébként is el fog hasalni, emiatt általában nincs arra szükség, hogy ezt külön teszteljük. Más kérdés, hogy ha egy metódus dobhat kivételt, akkor azt is tesztelni kell, hogy tényleg dog-e kivételt, amikor kell dobnia.

Ami alapján érdemes egyébként a teszt eseteket megírni az szerintem az, hogy 

- mi a jó útvonal, 
- mi történik ha a jó útvonal futása közben valami hiba keletkezik (metódusok hívása nem az elvárt eredményt adja)
- ha vannak a metódusnak paraméterei, akkor milyen különböző inputokra hogyan reagál

Ezekre külön-külön teszteket szoktunk írni.